draft-ietf-moq-transport.md

```diff
@@ -2100,6 +2145,148 @@ SUBSCRIBE_DONE, as defined below:
 * Too Far Behind - The publisher's queue of objects to be sent to the given
   subscriber exceeds its implementation defined limit.
 
+## PUBLISH {#message-publish}
+
+The publisher sends the PUBLISH control message to initiate a subscription to a
+track. The receiver verifies the publisher is authorized to publish this track.
+
+~~~
+PUBLISH Message {
+  Type (i) = 0x6,
+  Length (i),
+  Request ID (i),
+  Track Namespace (tuple),
+  Track Name Length (i),
+  Track Name (..),
+  Track Alias (i),
+  Group Order (8),
+  ContentExists (8),
+  [Largest (Location),]
+  Forward (8),
+  Number of Parameters (i),
+  Parameters (..) ...,
+}
```

Comment:
I think we are missing Publisher Priority specification . Today a publisher can suggest its priority in SubscribeOk though.

Reply:
> Today a publisher can suggest its priority in SubscribeOk though.

Discussed offline, no it can't, but you can file an issue if you want it to?